index.js

```diff
@@ -86,7 +86,7 @@ function RingPop(options) {
     this.isReady = false;
 
     this.debugFlags = {};
-    this.joinSize = 3;              // join fanout
+    this.joinSize = options.joinSize;
```

Comment:
it's defaulted in `join_cluster.js` which is a bit odd, but since `index.js` is the top-level ringpop object it's a temporary holding place for these values.